apps/admin-x-activitypub/src/components/Activities.tsx

```diff
@@ -11,6 +11,7 @@ import MainNavigation from './navigation/MainNavigation';
 import ViewProfileModal from './global/ViewProfileModal';
 
 import getUsername from '../utils/get-username';
+import stripHtml from '../utils/strip-html';
```

Comment:
How come we need to do this if we are sanitizing on the backend?

Reply:
@mike182uk  We have two different use cases:
1. Keeping HTML tags and classes and then rendering that HTML nicely formatted (e.g. post content when shown in Feed layout or when shown in the drawer)
2. Stripping all of the formatting and showing just plain text (e.g. post content when shown in Inbox layout, screenshot attached)

Is there a better way to achieve this?

<img width="508" alt="image" src="https://github.com/user-attachments/assets/f42ca8bd-6623-4c2f-8d3f-1667ffca6cb5">